External/FEXCore/Source/Interface/IR/IR.json

```diff
@@ -1109,31 +1100,46 @@
       ]
     },
 
-    "UCVTF": {
+    "FCVTFromGPR_U": {
```

Comment:
To go with the VCVT comment below, FCVT is also opaque.

Perhaps `Float_FromGPR_U`